modules/installation-network-user-infra.adoc

```diff
@@ -98,7 +98,7 @@ the Cluster Version Operator on port `9099`.
 
 |TCP/UDP
 |`30000`-`32767`
-|Kubernetes NodePort
+|Kubernetes `NodePort` service
```

Comment:
Similar to `type=LoadBalancer`, this is a `Service` object with `spec.type=NodePort`.

However, what this mean here is that the listed ports must be available, because it reflects the port range used on each node/worker for services of the above type.